### ISSUE TITLE
I have enabled FillHandle to true but still unable to drag cell value from one row to other row.

### ISSUE THREAD

**Issue by jainvikas82** [ag-grid/ag-grid#4401]
**I'm submitting a ...**  (check one with "x")
```
[] bug report => see 'Providing a Reproducible Scenario'
[] feature request => do not use Github for feature requests, see 'Customers of AG Grid'
[] support request => see 'Requesting Community Support'
```

**Customers of AG Grid**

If you are a customer you are entitled to use the AG Grid's customer support system (powered by Zendesk). Please use that channel for guaranteed response from the AG Grid team with regards bugs, feature requests and support.

**Requesting Community Support**

If you are not a customer of AG Grid, ag-grid staff will label your issue as managed-by-the-community. This means that AG Grid staff is not going to be actively looking into it and it will get closed if inactive for more than one month. The community is welcome to help with this question/support issue.

**Providing a Reproducible Scenario**
Accepted reproducible scenarios are
- A description of the detailed steps to reproduce your behaviour in one of our examples in the docs.
- A plunker

If you decide to send us a plunkr, from any example in our website use the plunkr button in there to fork your own code by following the steps below:

- Select the framework that is appropriate to you from the drop-down
- Open it in plunker. (Use the button plunker in our example)
- Add your changes so that the behaviour is reproduced
- Save and Freeze the plunker(On the top left corner)
- Send us the link to the plunker(You can copy the URL from the browser)

If reporting a bug make sure to state.

Current behaviour.
Expected behaviour. If possible back this up with our docs/examples if possible

**Current behavior**

**Expected behavior**

**Please tell us about your environment:**

* **AG Grid version:** X.X.X

* **Browser:**

* **Language:** [all | TypeScript X.X | ES6/7 | ES5]

**Comment by makinggoodsoftware** [comment]
Hi,

Can you provide a reproducible scenario using one of our examples or your source code in a plunker and we will investigate?

Accepted reproducible scenarios are
- A description of the detailed steps to reproduce your behaviour in one of our examples in the docs.
- A plunker

If you decide to send us a plunkr, from any example in our website use the plunkr button in there to fork your own code by following the steps below:

- Select the framework that is appropriate to you from the drop-down
- Open it in plunker. (Use the button plunker in our example)
- Add your changes so that the behaviour is reproduced
- Save and Freeze the plunker(On the top left corner)
- Send us the link to the plunker(You can copy the URL from the browser)

If reporting a bug make sure to state.

Actual behaviour.
Expected behaviour. If possible back this up with our docs/examples if possible

---

Note that this issue is flagged with waiting-for-repro.

If you are an ag-Grid Enterprise Customer, please raise this in our Zendesk support system - contact info@ag-grid.com for access. This is our primary channel for Support.

**Comment by jainvikas82** [comment]
Hello,

I have used the same code base from the below link but drag cell value functionality is not working. What am I missing here?
https://www.ag-grid.com/angular-grid/range-selection-fill-handle/


`import { Component } from '@angular/core';
import { HttpClient } from '@angular/common/http';
import 'ag-grid-enterprise';
import 'ag-grid-community/dist/styles/ag-grid.css';
import 'ag-grid-community/dist/styles/ag-theme-alpine.css';

@Component({
  selector: 'my-app',
  template: `<div class="example-wrapper">
    <div style="margin-bottom: 5px">
      <label>Axis: </label>
      <button
        class="ag-fill-direction xy selected"
        (click)="fillHandleAxis('xy')"
      >
        xy
      </button>
      <button class="ag-fill-direction x" (click)="fillHandleAxis('x')">
        x only
      </button>
      <button class="ag-fill-direction y" (click)="fillHandleAxis('y')">
        y only
      </button>
    </div>

    <ag-grid-angular
      #agGrid
      style="width: 100%; height: 100%;"
      id="myGrid"
      class="ag-theme-alpine"
      [columnDefs]="columnDefs"
      [defaultColDef]="defaultColDef"
      [enableRangeSelection]="true"
      [enableFillHandle]="true"
      [rowData]="rowData"
      (gridReady)="onGridReady($event)"
    ></ag-grid-angular>
  </div>`,
})
export class AppComponent {
  private gridApi;
  private gridColumnApi;

  private columnDefs;
  private defaultColDef;
  private rowData: [];

  constructor(private http: HttpClient) {
    this.columnDefs = [
      {
        field: 'athlete',
        minWidth: 150,
      },
      {
        field: 'age',
        maxWidth: 90,
      },
      {
        field: 'country',
        minWidth: 150,
      },
      {
        field: 'year',
        maxWidth: 90,
      },
      {
        field: 'date',
        minWidth: 150,
      },
      {
        field: 'sport',
        minWidth: 150,
      },
      { field: 'gold' },
      { field: 'silver' },
      { field: 'bronze' },
      { field: 'total' },
    ];
    this.defaultColDef = {
      flex: 1,
      minWidth: 100,
      editable: true,
    };
  }

  fillHandleAxis(direction) {
    var buttons = Array.prototype.slice.call(
      document.querySelectorAll('.ag-fill-direction')
    );
    var button = document.querySelector('.ag-fill-direction.' + direction);
    buttons.forEach(function (btn) {
      btn.classList.remove('selected');
    });
    button.classList.add('selected');
    this.gridApi.setFillHandleDirection(direction);
  }

  onGridReady(params) {
    this.gridApi = params.api;
    this.gridColumnApi = params.columnApi;

    this.http
      .get('https://www.ag-grid.com/example-assets/olympic-winners.json')
      .subscribe((data) => {
        this.rowData = data;
      });
  }
}`


![Code](https://user-images.githubusercontent.com/80791010/112768971-ac421d80-8fec-11eb-9ca6-9a05fdb61851.PNG)

**Comment by makinggoodsoftware** [comment]
HI,

If you fork the plunker from the docs and add a link here with a repro of your issue we will take a look

Thanks for your understanding!

**Comment by makinggoodsoftware** [comment]
Hi,

 

This ticket has been flagged as managed-by-community/waiting-for-repro for a while now and there has not been any activity, to help us tidy up we are going to close it.

 

Thanks

**Comment by kaichii** [comment]
Hi! Any workaround to fix it?

**Comment by StephenCooper** [comment]
Hi @kaichii , can you provide a reproduction of the issue because we have not been able to reproduce the failure and this example is working as expected https://www.ag-grid.com/angular-data-grid/range-selection-fill-handle/#example-fill-handle

**Comment by kaichii** [comment]
@StephenCooper  Thanks for your quick reply! In version 28, it works, but `enableRangeSelection` is the same as `enableFillHandle` in version 27, this is [online example](https://plnkr.co/edit/HnMkc4Z2aTg0IJAn?open=index.jsx).

**Comment by StephenCooper** [comment]
@kaichii Ok so the issue is with the changing of the value. This is a known limitation that the `enableFillHandle` is not a reactive property. 

There is a workaround that I have added to a fork of your plunker that recreates the grid when you change the value. https://plnkr.co/edit/erMhYLj1U42Kzkw4?open=index.jsx 

**However, you will have to evaluate if this is appropriate for your application as recreating the grid results in grid state being reset to its initial values.**

We have added this to our backlog and we are tracking it with the following reference and description:
**AG-7050 Make enableFillHandle a reactive property**
 

**Now that this issue is recorded in our backlog, we will soon close the GitHub issue because we will not update it when this bug is fixed in a future version of AG Grid. The best way to get an update on this item is to sign up for AG Grid new release notifications. Once a new AG Grid version is released, look for the item reference above (AG-7050) on the changelog page to see if it's resolved in that version:
https://www.ag-grid.com/ag-grid-changelog/**
